Realm.Shared/RealmList.cs

```diff
@@ -162,13 +162,13 @@ public int Count
                     throw new ArgumentOutOfRangeException();
                 }
 
-                var linkedRowPtr = _listHandle.Get((IntPtr)index);
+                var linkedRowPtr = _listHandle.Get((IntPtr)index, _realm.SharedRealmHandle);
```

Comment:
Here we're using `LinkView` though, and I couldn't find a way to extract the realm from that one 😞